tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/ProcessStartTests/ProcessStartTests.cs

```diff
@@ -46,23 +46,29 @@ public ProcessStartTests(ITestOutputHelper output)
         [Trait("RunOnWindows", "True")]
         public void IntegrationDisabled()
         {
-            const int totalSpanCount = 5;
+            const int totalSpanCount = 5; // we actually expect no spans when disabled
             const string expectedOperationName = "command_execution";
 
             SetEnvironmentVariable($"DD_TRACE_{nameof(IntegrationId.Process)}_ENABLED", "false");
 
             using var telemetry = this.ConfigureTelemetry();
             using var agent = EnvironmentHelper.GetMockAgent();
             using var process = RunSampleAndWaitForExit(agent);
-            var spans = agent.WaitForSpans(totalSpanCount, returnAllOperations: true);
+            var spans = agent.WaitForSpans(totalSpanCount, 2_000, returnAllOperations: true); // 2 seconds - no spans expected
 
             Assert.Empty(spans.Where(s => s.Name.Equals(expectedOperationName)));
             telemetry.AssertIntegrationDisabled(IntegrationId.Process);
         }
 
         private async Task RunTest(string metadataSchemaVersion)
         {
-            const int expectedSpanCount = 5;
+            int expectedSpanCount = 5;
+            // 3 on non-windows because of SecureString
+            if (!EnvironmentTools.IsWindows())
+            {
+                expectedSpanCount = 3;
+            }
+
```

Comment:
Same as `AgentMalfunctionTests` (use the same sample) there is only 3 spans in the snapshots for Linux/OSX and code comments seem to confirm that as well

Reply:
nit: I would use a ternary